lib/strong_migrations/migration.rb

```diff
@@ -119,6 +119,23 @@ def method_missing(method, *args, &block)
             raise_error :change_column_null,
               code: backfill_code(table, column, default)
           end
+        when :add_foreign_key
+          from_table, to_table, options = args
+          options ||= {}
+          validated = options.fetch(:validate) { true }
+
+          five_point_two = Gem::Version.new("5.2")
+          if postgresql? && ActiveRecord.version < five_point_two
```

Comment:
I feel like this is a better way to do gem version comparisons than boolean operations on `ActiveRecord::VERSION`

Reply:
Please keep it consistent for now.

Reply:
See #74